set.mm

```diff
@@ -775535,6 +775576,12 @@ additional condition (deduction form).  See ~ ralbidc for a more
       ( wa wrex anim1i r19.41v sylibr ) ACGBDEHZCGBCGDEHALCFIBCDEJK $.
   $}
 
+  $( Two ways of expressing "at most one" element.  (Contributed by Zhi Wang,
+     19-Sep-2024.) $)
+  rmotru $p |- ( E* x x e. A <-> E* x e. A T. ) $=
+    ( cv wcel wmo wtru wa wrmo trud pm4.71i mobii df-rmo bitr4i ) ACBDZAENFGZAE
+    FABHNOANFNIJKFABLM $.
```

Comment:
```suggestion
    ( cv wcel wmo wtru wa wrmo tru biantru mobii df-rmo bitr4i ) ACBDZAENFGZAEF
    ABHNOAFNIJKFABLM $.
```

Reply:
Similar statements and proofs are possible with `E.` and `E!` and all three deserve to be in Main, IMO.  Doable in a future PR, and by you or others, as you prefer.

Reply:
DONE. Not moved to main yet.